LayoutTests/http/tests/webgpu/webgpu/api/operation/command_buffer/programmable/state_tracking-expected.txt

```diff
@@ -1,3 +1,22 @@
+CONSOLE MESSAGE: Unhandled Promise Rejection: OperationError: popErrorScope failed
```

Comment:
these pass but output additional console logs, I'm not sure how to remove the console logs so I just added them to the expectations